src/api/props.js

```diff
@@ -1,15 +1,16 @@
 import { renderer as $renderer } from '../util/symbols';
 import assign from '../util/assign';
 
-function get (elem) {
+function get(elem) {
   const props = {};
-  for (let key in elem.constructor.props) {
+  Object.keys(elem.constructor.props).forEach((key) => {
```

Comment:
for...in loop are being yelled at by airbnb
in case of the loop there changing it on Object.keys breaks some tests for some reason, not sure why and should we invest time to fix it now.